### ISSUE TITLE
Version 1.11 w/ Festering Skys Feature State Tracker

### ISSUE THREAD

**Issue by paulgrove** [Mad-Mods-Phoenix-Point/AssortedAdjustments#5]
There are a lot of features in this mod, I will list them all here and mark which are confirmed working or confirmed broken - Separate issues will be raised for any that are broken, and linked to from here.

Please help and comment here if you can verify a feature does or doesn't work!

There are accounts of differences between Epic Game Store and Steam for which features are working.

I am updating this report so that the platform is also included.

I only have the EGS (Epic Game Store) version please comment below with updates to this table.

A blank cell indicates that no report has been received - the feature may or may not work.

## Preset

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| BalancePresetId | Works | | |
| BalancePresetState | Works | | |

## General

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| DisableRightClickMove | **Works** | | |
| DisableRocksAtBases | **Works** | | |
| EnableSmartEvacuation | **Works** | | |
| EnableSmartBaseSelection | **Works** | | |
| EnableScrapAircraft | **Works** | | |

## UX Enhancements

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| EnableExtendedAgendaTracker | **Works** | | |
| AgendaTrackerShowVehicles | **Works** | | |
| AgendaTrackerShowExcavations | **Works** | | |
| AgendaTrackerShowBaseDefenses | **Works** | | |
| AgendaTrackerHideStatusBar | **Works** | | |
| AgendaTrackerAddMissionObjective | **Works** | | |

## UI Enhancements

Be sure to disable the broken items here if you intend to play with these features - they are not benign and cause gameplay issues.

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| EnableUIEnhancements | **Partial Functionality** | | |
| ShowDetailedResearchAndProduction | **Broken** #8 | | |
| ShowRecruitInfoInsideZoneTooltip | **Broken** #6 | | |
| ShowExtendedBaseInfo | **Broken** #7 | | |
| ShowExtendedHavenInfo | **Works** | | |
| PersistentClassFilter | **Works** | | |
| PersistentClassFilterInitDisabled | **Works** | | |
| HideSiteAddons | **Works** | | |
| CompressPopulationBar | **Works** | | |
| HidePopulationBar | **Works** | | |
| BetterObjectives | **Works** | | |

## Limited War

Untested but comment by @sadhukar suggests this feature should work.

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| EnableLimitedWar | | | |
| LimitFactionAttacksToZones | | | |
| LimitPandoranAttacksToZones | | | |
| AttacksRaiseHavenAlertness | | | |
| AttacksRaiseFactionAlertness | | | |
| StopOneSidedWar | | | |
| DisablePandoranAttacksOnPhoenixBases | | | |
| StopAttacksWhileDefendingPandoransThreshold | | | |
| GlobalAttackLimit | | | |
| FactionAttackLimit | | | |
| UseDifficultyDrivenLimits | | | |

## Item Unlocks

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| UnlockItemsByResearch | | | |
| UnlockPhoenixEliteGear | | | |
| UnlockLivingWeapons | | | |
| UnlockLivingArmor | | | |
| UnlockIndependentAmmunition | **Works** | | |
| UnlockIndependentWeapons | **Works** | | |
| UnlockIndependentArmor | **Works** | | |

## Recruit Generation

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| EnableCustomRecruitGeneration | **Works** | | |
| RecruitGenerationCount | **Works** | | |
| IgnoreRngFactorsForHavenRecruitGeneration | **Works** | | |
| RecruitIntervalCheckDays | **Works** | | |
| RecruitGenerationHasArmor | **Works** | | |
| RecruitGenerationHasWeapons | **Works** | | |
| RecruitGenerationHasConsumableItems | **Works** | | |
| RecruitGenerationHasInventoryItems | **Works** | | |
| RecruitGenerationCanHaveAugmentations | **Works** | | |
| EnhanceSpecialRecruits | **Works** | | |

## Difficulty

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| EnableDifficultyOverrides | **Works** | | |
| DifficultyOverrideStartingSupplies | **Works** | | |
| DifficultyOverrideStartingMaterials | **Works** | | |
| DifficultyOverrideStartingTech | **Works** | | |
| DifficultyOverrideSoldierSkillPointsPerMission | **Works** | | |
| DifficultyOverrideExpConvertedToSkillpoints | **Works** | | |
| DifficultyOverrideMinPopulationThreshold | | | |
| DifficultyOverrideStarvationDeathsPart | | | |
| DifficultyOverrideStarvationMistDeathsPart | | | |
| DifficultyOverrideStarvationDeathsFlat | | | |
| DifficultyOverrideStarvationMistDeathsFlat | | | |
| DifficultyOverrideDisableDeathByStarvation | | | |
| DifficultyOverrideMistExpansionRate | | | |

## Progression

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| EnableExperienceToSkillpointConversion | | | |
| XPtoSPAddToPersonalPool | | | |
| XPtoSPAddToFactionPool | | | |
| XPtoSPConversionMultiplier | | | |

## Return Fire

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| EnableReturnFireAdjustments | | | |
| ReturnFireAngle | | | |
| ReturnFireLimit | | | |
| NoReturnFireWhenSteppingOut | | | |
| EmphasizeReturnFireHint | | | |

Mission Adjustments

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| EnableMissionAdjustments | **Works** | | |
| MaxPlayerUnitsAdd | **Works** | | |
| AlwaysRecoverAllItemsFromTacticalMissions | **Works** | | |
| EnablePlentifulItemDrops | **Works** | | |
| ItemDestructionChance |  **Works** | | |
| AllowWeaponDrops | **Works** | | |
| FlatWeaponDestructionChance | | | |
| HealthBasedWeaponDestruction | | | |
| AllowArmorDrops | **Works** | | | 
| FlatArmorDestructionChance | | | |

## Cinematics

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| SkipIntroLogos | **Works** | | | 
| SkipIntroMovie | **Works** | | | 
| SkipLandingSequences | **Works** | | | 

## Facility Modifications

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| EnableFacilityAdjustments | **Works** | | | 
| MedicalBayBaseHeal | **Works** | | | 
| LivingQuartersBaseStaminaHeal | **Works** | | | 
| VehicleBayAircraftHealAmount | **Works** | | | 
| VehicleBayVehicleHealAmount | **Works** | | | 
| MutationLabMutogHealAmount | **Works** | | | 
| TrainingFacilityBaseExperienceAmount | **Works** | | | 
| TrainingFacilitiesGenerateSkillpoints | **Works** | | | 
| TrainingFacilityBaseSkillPointsAmount | **Works** | | | 
| FabricationPlantGenerateProductionAmount | **Works** |  | |
| ResearchLabGenerateResearchAmount | **Works** | | | 
| FoodProductionGenerateSuppliesAmount | **Works** | | | 
| BionicsLabGenerateResearchAmount | **Works** | | | 
| MutationLabGenerateMutagenAmount | **Works** | | | 
| FabricationPlantGenerateMaterialsAmount | **Works** | | | 
| ResearchLabGenerateTechAmount | **Works** | | | 

## Soldiers

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| EnableSoldierAdjustments | **Works** | | | 
| MaxAugmentations | **Works** | | | 
| PersonalAbilitiesCount | **Works** | Broken | | 
| MaxStrength | **Works** | | | 
| MaxWill | **Works** | | | 
| MaxSpeed | **Works** | | | 
| Stamina | **Works** | | | 
| TiredStatusStaminaBelow | **Works** | | | 
| ExhaustedStatusStaminaBelow | **Works** | |  |

## Abilities

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| EnableAbilityAdjustments | | | |

## Items

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| EnableItemAdjustments | | | |

## Vehicles

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| EnableVehicleAdjustments | **Mostly Broken** #9 | | | 
| AircraftBlimpSpeed | **Broken** #9 | | | 
| AircraftThunderbirdSpeed | **Broken** #9 | | | 
| AircraftManticoreSpeed | **Broken** #9 | | | 
| AircraftHeliosSpeed | **Broken** #9 | | | 
| AircraftBlimpSpace | **Broken** #9 | | | 
| AircraftThunderbirdSpace | **Broken** #9 | | | 
| AircraftManticoreSpace | **Works** | | | 
| AircraftHeliosSpace | **Broken** #9 | | | 
| AircraftBlimpRange | **Broken** #9 | | | 
| AircraftThunderbirdRange | **Broken** #9 | | | 
| AircraftManticoreRange | **Broken** #9 | | | 
| AircraftHeliosRange | **Broken** #9 | | | 
| OccupyingSpaceMutog | **Broken** #9 | | | 
| OccupyingSpaceArmadillo | **Broken** #9 | | | 
| OccupyingSpaceScarab | **Broken** #9 | | | 
| OccupyingSpaceAspida | **Broken** #9 | | | 
| AmmoArmadillo | **Broken** #9 | | | 
| AmmoScarab | **Broken** #9 | | | 
| AmmoAspida | **Broken** #9 | | | 

## Economy

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| EnableEconomyAdjustments | **Works**  | | |
| ResourceMultiplier | **Works**  | | |
| ScrapMultiplier | **Works**  | | |
| CostMultiplier | **Works** || | |

## Events

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| DisableAmbushes | | | |
| RetainAmbushesInsideMist | | | |
| DisableNothingFound | | | |

## Pause and Center

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| PauseOnDestinationSet | **Works** | |  |
| PauseOnExplorationSet | **Works** | | | 
| PauseOnRecruitsGenerated | **Works** | | | 
| PauseOnHealed | **Works** | | | 
| CenterOnHavenRevealed | | | |
| CenterOnExcavationComplete | **Works** | | | 

## Dev stuff

| Feature | EGS 1.11 | Steam 1.11 | Gog 1.11 |
| --- | :---: | :---: | :---: |
| Debug | | | |
| DebugDevKey | | | |
| PresetStateHash | | | |

**Comment by mad2342** [comment]
Hey paulgrove ,
Thx for your efforts! Just wanted to say that at some point i will return to this (dlc and patch isn't even avail to me(gog) and i'm a bit busy atm) and this will probably help alot to get things done faster...
Cheers, Mad

**Comment by sadhukar** [comment]
Hide Site Addons, Better Objectives - works
Limited War - unable to test, but I would think it works
Recruit Generation - Tested on my non-dlc save, works
Vehicles - all broken

**Comment by sadhukar** [comment]
> Hey paulgrove ,
> Thx for your efforts! Just wanted to say that at some point i will return to this (dlc and patch isn't even avail to me(gog) and i'm a bit busy atm) and this will probably help alot to get things done faster...
> Cheers, Mad

Hey @mad2342 

I will literally gift you the dlc on GoG if it means you can update your mod soon!

**Comment by Potato298** [comment]
If PersonalAbilitiesCount is the one that lets you crank up the number abilities to 7 then it does not work. At least for me.

**Comment by jhonNow** [comment]
Epic version Modnix 2.5.7 and Assorted Adjustment 1.10.1.4
EnableDifficultyOverrides
 DifficultyOverrideStartingSupplies
 DifficultyOverrideStartingMaterials
 DifficultyOverrideStartingTech

working

**Comment by paulgrove** [comment]
> If PersonalAbilitiesCount is the one that lets you crank up the number abilities to 7 then it does not work. At least for me.

It worked for me - but I had to start a new game to see the effect (I set it to 5 in my test).

~~I can test again if you insist it isn't working.~~

I tested again - it's working - I started a new game again though:
![image](https://user-images.githubusercontent.com/184906/119778021-dade5680-bebe-11eb-9179-cc1123874875.png)

I suspect an old game would work - but you have to wait for a new unit to be generated - all existing units and the units at havens and on your recruitment screen will already have their abilities decided if you simply load a save.

**Comment by jhonNow** [comment]
> > If PersonalAbilitiesCount is the one that lets you crank up the number abilities to 7 then it does not work. At least for me.
> 
> It worked for me - but I had to start a new game to see the effect (I set it to 5 in my test).
> 
> ~I can test again if you insist it isn't working.~
> 
> I tested again - it's working - I started a new game again though:
> ![image](https://user-images.githubusercontent.com/184906/119778021-dade5680-bebe-11eb-9179-cc1123874875.png)
> 
> I suspect an old game would work - but you have to wait for a new unit to be generated - all existing units and the units at havens and on your recruitment screen will already have their abilities decided if you simply load a save.

I agree Paul started a new game and mine was also at 5 .

**Comment by Potato298** [comment]
Weird, i'll have to try again. I tried both going to an old save from a day before the patch, waiting for the new recruits to be available and they all had 3 abilities so i went and made a new game and the same thing happened.

**Comment by paulgrove** [comment]
@Potato298 note you also have to have EnableSoldierAdjustments enabled.

**Comment by Potato298** [comment]
[AssortedAdjustements.txt](https://github.com/Mad-Mods-Phoenix-Point/AssortedAdjustments/files/6563996/AssortedAdjustements.txt)

Here is my config for AA. EnableSoldierAdjustments was set on true and i tried setting PersonalAbilitiesCount to both 5 and 7 in and testing them on an existing save and a new game after waiting for the new recruits to spawn, still not working. I had Select Class Traits and tried deleting it just in case and nope, still not working. I guess i'm cursed

**Comment by jhonNow** [comment]
> [AssortedAdjustements.txt](https://github.com/Mad-Mods-Phoenix-Point/AssortedAdjustments/files/6563996/AssortedAdjustements.txt)
> 
> Here is my config for AA. EnableSoldierAdjustments was set on true and i tried setting PersonalAbilitiesCount to both 5 and 7 in and testing them on an existing save and a new game after waiting for the new recruits to spawn, still not working. I had Select Class Traits and tried deleting it just in case and nope, still not working. I guess i'm cursed

Ok this is what i did before starting a new game 
deleted all inside mygames/phoenixpoint/mods 

then in C:\Program Files\Epic Games\PhoenixPoint\PhoenixPointWin64_Data\Managed i deleted all the content 

Then in epic launcher found the game and did "verify" the game (so i got a fresh managed map) 
after that i installed modnix and the assorted adjustment mod .

After that my game worked and i could make the above described parts working

**Comment by jarnomikkola1** [comment]
> After that my game worked and i could make the above described parts working

Just to be %&/# particular about this;
So you are saying that it works in the EGS version v1.11 of the game, with the Assorted Adjustment 1.10.1.4 version, and Modnix 2.5.7 ? 
I ask cause I use a little different setup, a steam based game, and not the personal perk number, nor the attribute adjustments, or the start resource adjustments work in my setup. 
But some of the other tweaks do work.

**Comment by Potato298** [comment]
> > [AssortedAdjustements.txt](https://github.com/Mad-Mods-Phoenix-Point/AssortedAdjustments/files/6563996/AssortedAdjustements.txt)
> > Here is my config for AA. EnableSoldierAdjustments was set on true and i tried setting PersonalAbilitiesCount to both 5 and 7 in and testing them on an existing save and a new game after waiting for the new recruits to spawn, still not working. I had Select Class Traits and tried deleting it just in case and nope, still not working. I guess i'm cursed
> 
> Ok this is what i did before starting a new game
> deleted all inside mygames/phoenixpoint/mods
> 
> then in C:\Program Files\Epic Games\PhoenixPoint\PhoenixPointWin64_Data\Managed i deleted all the content
> 
> Then in epic launcher found the game and did "verify" the game (so i got a fresh managed map)
> after that i installed modnix and the assorted adjustment mod .
> 
> After that my game worked and i could make the above described parts working

Modnix 2.5.7 and Assorted Adjustment 1.10.1.4 here. Tried a clean install like you described and still not working.


> > After that my game worked and i could make the above described parts working
> 
> Just to be %&/# particular about this;
> So you are saying that it works in the EGS version v1.11 of the game, with the Assorted Adjustment 1.10.1.4 version, and Modnix 2.5.7 ?
> I ask cause I use a little different setup, a steam based game, and not the personal perk number, nor the attribute adjustments, or the start resource adjustments work in my setup.
> But some of the other tweaks do work.

@jarnomikkola1 Maybe it's a Steam Version issue then?

**Comment by paulgrove** [comment]
As a point of data, I am using the Epic Version of the game, with Modnix 2.5.7, Assorted Adjustments 1.0.10.1.4 and Phoenix Point ver 1.11.69525

I created a minimal configuration with everything disabled, and then enabled only this feature and it worked.  I am now playing a game with most working features enabled and with PersonalAbilitiesCount set to 5 and it is definitely working.

I just thought to check, after the upgrade to 1.11 with the new DLC I had to reconfigure Modnix as is was showing as not injected, is it showing as injected in the top left of the modnix ui here:
![image](https://user-images.githubusercontent.com/184906/120087516-b366d400-c0e0-11eb-81bd-15d7f4c604d7.png)

Also here is my config:
[AssortedAdjustments.txt](https://github.com/Mad-Mods-Phoenix-Point/AssortedAdjustments/files/6565462/AssortedAdjustments.txt)

**Comment by jhonNow** [comment]
> > > [AssortedAdjustements.txt](https://github.com/Mad-Mods-Phoenix-Point/AssortedAdjustments/files/6563996/AssortedAdjustements.txt)
> > > Here is my config for AA. EnableSoldierAdjustments was set on true and i tried setting PersonalAbilitiesCount to both 5 and 7 in and testing them on an existing save and a new game after waiting for the new recruits to spawn, still not working. I had Select Class Traits and tried deleting it just in case and nope, still not working. I guess i'm cursed
> > 
> > 
> > Ok this is what i did before starting a new game
> > deleted all inside mygames/phoenixpoint/mods
> > then in C:\Program Files\Epic Games\PhoenixPoint\PhoenixPointWin64_Data\Managed i deleted all the content
> > Then in epic launcher found the game and did "verify" the game (so i got a fresh managed map)
> > after that i installed modnix and the assorted adjustment mod .
> > After that my game worked and i could make the above described parts working
> 
> Modnix 2.5.7 and Assorted Adjustment 1.10.1.4 here. Tried a clean install like you described and still not working.
> 
> > > After that my game worked and i could make the above described parts working
> > 
> > 
> > Just to be %&/# particular about this;
> > So you are saying that it works in the EGS version v1.11 of the game, with the Assorted Adjustment 1.10.1.4 version, and Modnix 2.5.7 ?
> > I ask cause I use a little different setup, a steam based game, and not the personal perk number, nor the attribute adjustments, or the start resource adjustments work in my setup.
> > But some of the other tweaks do work.
> 
> @jarnomikkola1 Maybe it's a Steam Version issue then?

I use also same version in Epic 
Modnix 2.5.7, Assorted Adjustments 1.0.10.1.4 and Phoenix Point ver 1.11.69525

**Comment by jarnomikkola1** [comment]
Erhm;
Vehicles - All Broken #9
...
AircraftManticoreSpace - WORKS - in the EGS version !!! But only there.

**Comment by jhonNow** [comment]
> Erhm;
> Vehicles - All Broken #9
> ...
> AircraftManticoreSpace - WORKS - in the EGS version !!! But only there.

true works on egs

**Comment by paulgrove** [comment]
Please be explicit with the platform when you are reporting feature status, I have updated the table to reflect feature status across different platforms.

**Comment by mad2342** [comment]
https://github.com/Mad-Mods-Phoenix-Point/AssortedAdjustments/releases/tag/1.11.0.0
Fixed the most obvious problems, checked most critical features, they all work for me. Please note that this is not yet thoroughly tested and that i do not own FS yet.

**Comment by paulgrove** [comment]
Thanks @mad2342 I'll give it a try tonight, been enjoying the new DLC - wasn't sure at first, but now i'm getting into the meat of it I'm really starting to have some fun.

Can I donate a copy of Festering Skies to you?

**Comment by mad2342** [comment]
Hi paulgrove,
That's nice of you! I'm on GOG with the base game though, don't know if that's an option for you. And please note that i cannot guarantee any dev of AA, just to be perfectly honest with you!
If you still like to donate, i think in GOG you can order as a gift and email some info/code to someone else, i'd be happy: github@mad-mods.com
Best regards, Mad

**Comment by paulgrove** [comment]
Sent! and no pressure for any dev work, to be frank most feature I use are working, imo this mod has made the game so much more enjoyable for me, and I wouldn't probably be playing through it again without it.

**Comment by mad2342** [comment]
Thank you, i appreciate it!
Now let's see what FS has to offer... :-)

**Comment by sadhukar** [comment]
Hi guys,

So what features are working in your updated release and which aren't? Will they work for the steam version as well?

**Comment by mad2342** [comment]
Hi guys,
Thx again paulgrove for the effort you put in this. 
I finally tested this stuff with current game version on GOG, for me everything seems to be in order.
I have no idea why the Steam/Epic version would behave differently once Modnix and AA itself get loaded without errors as there are no indications that there's platform dependent code.
Closing this issue, feel free to open another if problems persist.
